app/javascript/components/charts/AmChartGauge.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+  <div class="am-chart--gauge">
+    <p 
+      v-if="title"
+      class="chart__title"
+    >
+      {{ title }}
+    </p>
+
+    <div class="chart__chart">
+      <div 
+        class="chart__svg"
+        :id="id"
+      />
+    </div>
+  </div>
+</template>
+
+<script>
+import * as am4core from "@amcharts/amcharts4/core"
+import * as am4charts from "@amcharts/amcharts4/charts"
+import am4themes_animated from "@amcharts/amcharts4/themes/animated"
+import ScrollMagic from 'scrollmagic'
+
+export default {
+  name: 'AmChartGauge',
+
+  props: {
+    id: {
+      required: true,
+      type: String
+    },
+    legendData: {
+      type: Array // [{ name: String, fill: Strin }]
+    },
+    target: {
+      type: Number
+    },
+    theme: {
+      default: '#000',
+      type: String,
+    },
+    title: {
+      String: Number
+    },
+    trigger: { 
+      type: String
+    },
+    value: {
+      required: true,
+      type: Number
+    }
+  },
+
+  date () {
+    return {
+      axis: {},
+      chart: {},
+    }
+  },
+
+  mounted () {
+    this.createChart()
+
+    am4core.useTheme(am4themes_animated)
+
+    if(this.trigger) {
+      this.scrollMagicHandlers()
+    }
+  },
+
+  methods: {
+    animation () {
+      console.log('animation')
+      new am4core.Animation(this.hand, {
+        property: "value",
+        to: this.value
+      }, 1000, am4core.ease.cubicOut).start()
+
+      new am4core.Animation(this.range, {
+        property: "endValue",
+        to: this.value
+      }, 1000, am4core.ease.cubicOut).start()
+    },
+
+    createChart () {
+      this.chart = am4core.create(this.id, am4charts.GaugeChart)
+      
+      this.chart.paddingBottom = 90
+      this.chart.paddingLeft = 0
+      this.chart.innerRadius = 100
+
+      this.createAxis()
+      this.createRange()
+      this.createHands()
+      this.createLegend()
+    },
+
+    createAxis () {
+      this.axis = this.chart.xAxes.push(new am4charts.ValueAxis());
+      this.axis.min = 0
+      this.axis.max = 100
+      this.axis.renderer.radius = 100
+      this.axis.strictMinMax = true
+      this.axis.renderer.line.strokeOpacity = 1
+      this.axis.renderer.labels.template.opacity = 0
+
+      this.axisInner = this.chart.xAxes.push(new am4charts.ValueAxis());
+      this.axisInner.min = 0
+      this.axisInner.max = 100
+      this.axisInner.renderer.radius = 60
+      this.axisInner.strictMinMax = true
+      this.axisInner.renderer.line .strokeOpacity = 1
```

Comment:
For some reason the space between `line` and `.strokeOpacity` isn't picked up by ESLint in my VSCode as an error, and it still seems to work with it. I would still fix this though